docs/fidesops/docs/guides/admin_ui.md

```diff
@@ -0,0 +1,101 @@
+# Administrator UI
+
+FidesOps provides a built-in control panel for managing privacy requests.
+
+![admin ui](../img/admin_ui.png)
+
+## Running Locally
+
+To get started, clone the [FidesOps repository](https://github.com/ethyca/fidesops/).
+
+### Creating the root user
+
+In the top-level `fidesops` directory, run `make user`.
+
+A series of prompts will walk you through creating a username and password. Passwords require 8 or more characters, upper and lowercase characters, a number, and a symbol. 
+
+This will create an Admin UI Root User that can be used to access the in-browser control panel, and additional [user endpoints](#managing-users).
```

Comment:
Should we be directing them to use the admin ui root user? I thought the original intent of the admin UI root user was just to create the first user. This user could then create other users, which would be the users to actually log in and manage requests.